packages/python/plotly/test_requirements/requirements_39_pandas_2_optional.txt

```diff
@@ -1,6 +1,6 @@
 requests==2.25.1
 tenacity==6.2.0
-pandas==2.0.1
+pandas==2.0.2
```

Comment:
Without this change, the protocol cannot be used in tests (for example, vaex test).